gems/pending/util/postgres_admin.rb

```diff
@@ -152,11 +186,11 @@ def self.gc(options = {})
     options = (options[:aggressive] ? GC_AGGRESSIVE_DEFAULTS : GC_DEFAULTS).merge(options)
 
     result = self.vacuum(options)
-    _log.info("Output... #{result}") if result.to_s.length > 0
+    $log.info("MIQ(#{self.name}.#{__method__}) Output... #{result}") if result.to_s.length > 0
 
     if options[:reindex]
       result = self.reindex(options)
-      _log.info("Output... #{result}") if result.to_s.length > 0
+      $log.info("MIQ(#{self.name}.#{__method__}) Output... #{result}") if result.to_s.length > 0
```

Comment:
These log lines were changed because we moved the file out of the rails app and don't include Vmdb::Logging anymore.